docs/userguide/storagedriver/aufs-driver.md

```diff
@@ -0,0 +1,194 @@
+single unified view. If a container is not running, its directory still exists
+but is empty. This is because containers are only mounted when they are running.
+
+Container metadata and various config files that are placed into the running
+container are stored in `/var/lib/contaienrs/<container-id>`. Files in this
+directory exist for all containers on the system, including ones that are
+stopped. However, when a container is running the container's log file are also
+in this directory.
+
+A container's thin writable layer is stored under
+`/var/lib/docker/aufs/diff/<container-id>`. This directory is stacked by AUFS as
+the containers top writable layer and is where all changes to the container are
+stored. The directory exists even if the container is stopped. This means that
+restarting a container will not lose changes made to it. Once a container is
+deleted this directory is deleted.
+
+Information about which image layers are stacked below a container's top
+writable layer is stored in the following file
+`/var/lib/docker/aufs/layers/<container-id>`. The command below shows that the
+container with ID `b41a6e5a508d` has 4 image layers below it:
+
+```bash
+$ cat /var/lib/docker/aufs/layers/b41a6e5a508dfa02607199dfe51ed9345a675c977f2cafe8ef3e4b0b5773404e-init
+91e54dfb11794fad694460162bf0cb0a4fa710cfa3f60979c177d920813e267c
+d74508fb6632491cea586a1fd7d748dfc5274cd6fdfedee309ecdcbc2bf5cb82
+c22013c8472965aa5b62559f2b540cd440716ef149756e7b958a1b2aba421e87
+d3a1f33e8a5a513092f01bb7eb1c2abf4d711e5105390a3fe1ae2248cfde1391
+```
+
+The image layers are shown in order. In the output above, the layer starting
+with image ID "d3a1..." is the image's base  layer. The image layer starting
+with "91e5..." is the image's topmost layer.
+
+
+## AUFS and Docker performance
+
+To summarize some of the performance related aspects already mentioned:
+
+- The AUFS storage driver is a good choice for PaaS and other similar use-cases where container density is important. This is because AUFS efficiently shares images between multiple running containers, enabling fast container start times and minimal use of disk space.
+
+- The underlying mechanics of how AUFS shares files between image layers and containers uses the systems page cache very efficiently.
+
+- The AUFS storage driver can introduce significant latencies into container write performance. This is because the first time a container writes to any file, the file has be located and copied into the containers top writable layer. These latencies increase and are compounded when these files exist below many image layers and the files themselves are large.
+
+One final point. Data volumes often provide the best and most predictable performance. This is because they bypass the storage driver and do not incur any of the potential overheads introduced union mounts and copy-on-write. For this reason, you may want to place heavy write workloads on data volumes.
```

Comment:
I reckon we should change this final para to - "One final point. Data volumes provide the best and most predictable performance. This is because they bypass the storage driver and do not incur any of the potential overheads introduced by thin provisioning and copy-on-write. For this reason, you may want to place heavy write workloads on data volumes."  This keeps it in line with the rest of the storage driver  docs.